torch/testing/_internal/common_methods_invocations.py

```diff
@@ -14268,6 +14289,58 @@ def ref_pairwise_distance(input1, input2):
                 "test_variant_consistency_jit",
             ),
         ),
+    ),
+    OpInfo(
+        "nn.functional.binary_cross_entropy",
+        sample_inputs_func=sample_inputs_binary_cross_entropy,
+        dtypes=floating_types(),
+        dtypesIfCUDA=floating_types_and(torch.float16, torch.bfloat16),
+        supports_out=False,
+        decorators=(
+            DecorateInfo(
```

Comment:
I guess this is technically unnecessary because this test is xfailed below?

Reply:
Yes, but without the decorator, the test would only complain about insufficient tolerances, which hides the actual issue. Thus if the source of the error is fixed, we need the decorator.